modules/dnn/test/test_model.cpp

```diff
@@ -7,68 +7,88 @@
 #include "npy_blob.hpp"
 namespace opencv_test { namespace {
 
-template<typename TString>
```

Comment:
Please avoid massive changes (looks like simple indentation)

Reply:
@dkurt, I am terribly sorry, did not notice that mess. I must have autoindented it by accident. Anyway, it is fixed now